### PR TITLE
Create kiro7.com.xml

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

File: src/chrome/content/rules/kiro7.com.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!--
+	Invalid certificate:
+		a135.kiro7.com
+		amp.kiro7.com
+		click.e.kiro7.com
+		image.e.kiro7.com
+		li.kiro7.com
+		sa135.kiro7.com
```
Comment (J0WI) [inline src/chrome/content/rules/kiro7.com.xml]: The page tries to load some scripts from there. I don't know if they are used somewhere. Do we care?

Comment (Bisaloo) [reply src/chrome/content/rules/kiro7.com.xml]: Hum, I don't see this.

Comment (J0WI) [reply src/chrome/content/rules/kiro7.com.xml]: @jeremyn could you double check?

Comment (jeremyn) [reply src/chrome/content/rules/kiro7.com.xml]: I'm not sure what I'm checking. https://sa135.kiro7.com has an invalid certificate, and I don't see any network requests to `sa135.kiro7.com` from https://www.kiro7.com/.

**Pull request merged**